packages/nimble-blazor/NimbleBlazor/Components/NimbleTableColumnEnumText.razor

```diff
@@ -0,0 +1,22 @@
+﻿@namespace NimbleBlazor
+@inherits NimbleTableColumn
+@attribute [CascadingTypeParameter(nameof(TKey))]
+@typeparam TKey
+
+<nimble-table-column-enum-text
```

Comment:
This doesn't have `@attributes="AdditionalAttributes"`, but it doesn't look like our other columns have that in Blazor either. Do you know if that's intentional? @atmgrifter00, perhaps you know?

Reply:
Yeah, that's not intentional. For now, we should have that on all of our public component APIs.